src/lib.rs

```diff
@@ -106,12 +108,13 @@ pub enum Response {
     FileSendChunkError { error: FileSendChunkError },
 
     Conference { conference: u32 },
+    ConferencePeerList { peers: Vec<PeerInfo> },
     ConferencePeerCount { count: u32 },
     ConferencePeerName { name: String },
     ConferencePeerPublicKey { public_key: String },
     IsOwnPeerNumber { is_own: bool, },
     ConferenceTitle { title: String, },
-    ChatList { list: Vec<u32> },
+    ChatList { conferences: Vec<ConferenceInfo> },
```

Comment:
ConferencePeerList vs ChatList? Why not ConferenceList?

Reply:
Name from the Tox API. I probably should change it.